app/services/api/v1x0.rb

```diff
@@ -0,0 +1,42 @@
+require 'services/api/v1x0/catalog'
```

Comment:
Does it make more sense to have each level require the children instead of loading everything here?

I'm thinking this file just requires the files at this directly level dynamically and then each child loads the files in their directory.  Or if this really just a require of all the ruby files under this directory could we achieve the same thing with a `Dir.glob` call.

@syncrou Do you have thoughts on this?

Reply:
@mkanoor If we are going with this method does it makes sense to replace all these with:

```ruby
Dir.glob(Pathname.new(__dir__).join("v1x0/**/*.rb")) do |file|
  require file.to_s
end
```